deploy/csi-daemonset.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       # the same.
       containers:
       - name: mayastor-csi
-        image: mayadata/mayastor-csi:develop
+        image: mayadata/mayastor:develop
```

Comment:
Do we run mayastor-csi also from the mayastor-binary?

Reply:
It would appear so. This and the previous comment refer to code originally part of #1045.